backend/src/app.controller.ts

```diff
@@ -7,6 +7,7 @@ export class AppController {
 
   @Get()
   getHello(): string {
+	this.appService.getGame();
```

Comment:
getHello is een voorbeeld functie van NestJS. die gaan we uiteindelijk niet gebruiken. deze code mag ook worden verplaatst binnen het 'game' mapje. in een game.gateway.ts file (als je t via websockets wil doen en niet via http)

Reply:
kan er voor nu nog blijven staan